EvtWeightTools/src/EvtWeightRecorder.cc

```diff
@@ -776,6 +777,84 @@ EvtWeightRecorder::record_lheScaleWeight(const LHEInfoReader * const lheInfoRead
   }
 }
 
+std::pair<std::vector<int>, std::vector<int>>
+EvtWeightRecorder::findHiggs(const LHEParticleCollection & lheParticles, std::vector<int> & exclude)
+{
+  int bestindex1(-1), bestindex2(-1);
+  TLorentzVector d1, d2;
+  std::vector<int> higgs;
+  for (unsigned int indx1=0; indx1<lheParticles.size(); indx1++)
+  {
+    if ( lheParticles[indx1].pdgId() == 25 )
+    {
+      higgs.push_back(indx1);
+      if ( higgs.size() == 2 ) break;
+      continue;
+    }
+    if ( indx1 == lheParticles.size()-1 ) break;
```

Comment:
I think this line is not needed, because the for loop two lines below won't be executed if `indx1 == lheParticles.size()-1`.